phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -1286,8 +1286,44 @@ public function test_resolve_theme_file_uris() {
 	public function test_get_resolved_theme_uris() {
 		$theme_json = new WP_Theme_JSON_Gutenberg(
 			array(
-				'version' => WP_Theme_JSON_Gutenberg::LATEST_SCHEMA,
-				'styles'  => array(
+				'version'  => WP_Theme_JSON_Gutenberg::LATEST_SCHEMA,
```

Comment:
I updated the current test to include fonts - hope you don't mind!

Reply:
:+1: cool thanks